db/migrate/20170315095936_update_persistent_volumes_parent_type.rb

```diff
@@ -0,0 +1,7 @@
+class UpdatePersistentVolumesParentType < ActiveRecord::Migration[5.0]
+  def change
+    PersistentVolume.all.each do |pv|
```

Comment:
batches:
when we know that there is thousand records it is nice to use 
find_each - by default it is splitting to batches by 1000 records.
and it looks like find_each is using find_in_batches as @cben is suggesting 👍